tests/queries/0_stateless/01768_array_product.reference

```diff
@@ -0,0 +1,16 @@
+Array product 	720
+28.799999999999997	Float64
+3.5	Float64
+20	Float64
+720
+0
+6
+15504.12105639681
```

Comment:
This result is not correct, because In the implementation, the type of [1,2,3,4,5,6] is Decimal64(8). 
So the value I get to multiple is 100000000,200000000 ,300000000,400000000,500000000,600000000 and the result is overflow, How can I resolve this problem?

Reply:
We can throw exception on overflow (by utilizing `mulOverflow`) or allow overflow (leave it implementation specific).